# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2016-11-24 11:34+0100\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../plugins/repoclosure.py:46
msgid "Display a list of unresolved dependencies for repositories"
msgstr ""

#: ../plugins/repoclosure.py:117
msgid "check packages of the given archs, can be specified multiple times"
msgstr ""

#: ../plugins/repoclosure.py:120
msgid "Specify repositories to check"
msgstr ""

#: ../plugins/repoclosure.py:122
msgid "Check only the newest packages in the repos"
msgstr ""

#: ../plugins/repoclosure.py:125
msgid "Check closure for this package only"
msgstr ""

#: ../plugins/versionlock.py:32
#, python-format
msgid "Unable to read version lock configuration: %s"
msgstr ""

#: ../plugins/versionlock.py:33
msgid "Locklist not set"
msgstr ""

#: ../plugins/versionlock.py:34
msgid "Adding versionlock on:"
msgstr ""

#: ../plugins/versionlock.py:35
msgid "Adding exclude on:"
msgstr ""

#: ../plugins/versionlock.py:36
msgid "Deleting versionlock for:"
msgstr ""

#: ../plugins/versionlock.py:37
msgid "No package found for:"
msgstr ""

#: ../plugins/versionlock.py:89
msgid "control package version locks"
msgstr ""

#: ../plugins/migrate.py:57
#, python-format
msgid "failed to parse package NEVRA from: %s"
msgstr ""

#: ../plugins/migrate.py:78
msgid "migrate yum's history, group and yumdb data to dnf"
msgstr ""

#: ../plugins/migrate.py:95
msgid "which kind of yum data migrate."
msgstr ""

#: ../plugins/migrate.py:111
msgid "Migrating history data..."
msgstr ""

#: ../plugins/migrate.py:222
msgid "Migrating groups data..."
msgstr ""

#: ../plugins/migrate.py:227
msgid "Execution of Yum failed. Could not retrieve installed groups."
msgstr ""

#: ../plugins/migrate.py:231
msgid "No groups to migrate from Yum"
msgstr ""

#: ../plugins/migrate.py:243
msgid "skipping."
msgstr ""

#: ../plugins/migrate.py:263
msgid "Migrating YUMDB data..."
msgstr ""

#: ../plugins/migrate.py:282
#, python-format
msgid "%s of %s not found"
msgstr ""

#: ../plugins/migrate.py:289
#, python-format
msgid "replacing unknown characters in %s of %s"
msgstr ""

#: ../plugins/migrate.py:294
msgid "DNFDB access denied"
msgstr ""

#: ../plugins/migrate.py:296
#, python-format
msgid "%s of %s migrated"
msgstr ""

#: ../plugins/migrate.py:300
#, python-format
msgid "%d YUMDB records found, %d migrated, %d skipped/preserved"
msgstr ""

#: ../plugins/repograph.py:54
msgid "Output a full package dependency graph in dot format"
msgstr ""

#: ../plugins/repograph.py:115
#, python-format
msgid "Nothing provides: '%s'"
msgstr ""

#: ../plugins/kickstart.py:58
msgid "Install packages defined in a kickstart file on your system"
msgstr ""

#: ../plugins/kickstart.py:63
msgid "kickstart file"
msgstr ""

#: ../plugins/kickstart.py:81
#, python-format
msgid "file cannot be parsed: %s"
msgstr ""

#: ../plugins/kickstart.py:98
#, python-format
msgid "No package %s available."
msgstr ""

#: ../plugins/kickstart.py:103
msgid "Nothing to do."
msgstr ""

#: ../plugins/show_leaves.py:56
msgid "New leaves:"
msgstr ""

#: ../plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr ""

#: ../plugins/snapper.py:54
msgid "creating snapshot"
msgstr ""

#: ../plugins/snapper.py:59
#, python-format
msgid "created snapshot %d"
msgstr ""

#: ../plugins/snapper.py:63
#, python-format
msgid "creating snapshot failed: %s"
msgstr ""

#: ../plugins/repomanage.py:44
msgid "Manage a directory of rpm packages"
msgstr ""

#: ../plugins/repomanage.py:52
msgid "Pass either --old or --new, not both!"
msgstr ""

#: ../plugins/repomanage.py:62
msgid "No files to process"
msgstr ""

#: ../plugins/repomanage.py:67
msgid "Could not open {}"
msgstr ""

#: ../plugins/repomanage.py:124
msgid "Print the older packages"
msgstr ""

#: ../plugins/repomanage.py:126
msgid "Print the newest packages"
msgstr ""

#: ../plugins/repomanage.py:128
msgid "Space separated output, not newline"
msgstr ""

#: ../plugins/repomanage.py:130
msgid "Newest N packages to keep - defaults to 1"
msgstr ""

#: ../plugins/repomanage.py:133
msgid "Path to directory"
msgstr ""

#: ../plugins/torproxy.py:31
msgid "Disabling torproxy plugin: cannot connect to the Tor network"
msgstr ""

#: ../plugins/rpm_conf.py:67
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr ""

#: ../plugins/rpm_conf.py:73
msgid "rpmconf plugin will not run in non-interactive mode"
msgstr ""

#: ../plugins/rpm_conf.py:86
msgid "ignoring sys.exit from rpmconf due to missing MERGE variable"
msgstr ""

#: ../plugins/rpm_conf.py:90
msgid "ignoring sys.exit from rpmconf due to missing file"
msgstr ""

#: ../plugins/leaves.py:32
msgid "List installed packages not required by any other package"
msgstr ""

#: ../plugins/system_upgrade.py:54
msgid "the color of the sky"
msgstr ""

#: ../plugins/system_upgrade.py:73
msgid "No new kernel packages were found."
msgstr ""

#: ../plugins/system_upgrade.py:75
msgid "Need a --releasever greater than the current system version."
msgstr ""

#: ../plugins/system_upgrade.py:77
msgid ""
"Download complete! Use 'dnf system-upgrade reboot' to start the upgrade."
msgstr ""

#: ../plugins/system_upgrade.py:79
msgid "Sorry, you need to use 'download --releasever' instead of '--network'"
msgstr ""

#: ../plugins/system_upgrade.py:112
msgid "--datadir: File exists"
msgstr ""

#: ../plugins/system_upgrade.py:257
msgid "The following boots appear to contain upgrade logs:"
msgstr ""

#: ../plugins/system_upgrade.py:267
msgid "-- no logs were found --"
msgstr ""

#: ../plugins/system_upgrade.py:282
msgid "Cannot find logs with this index."
msgstr ""

#: ../plugins/system_upgrade.py:305
msgid "Prepare system for upgrade to a new release"
msgstr ""

#: ../plugins/system_upgrade.py:314
msgid "save downloaded data to this location"
msgstr ""

#: ../plugins/system_upgrade.py:317
msgid "keep installed packages if the new release's version is older"
msgstr ""

#: ../plugins/system_upgrade.py:398
msgid "system is not ready for upgrade"
msgstr ""

#: ../plugins/system_upgrade.py:400
msgid "upgrade is already scheduled"
msgstr ""

#: ../plugins/system_upgrade.py:406
msgid "use 'dnf system-upgrade reboot' to begin the upgrade"
msgstr ""

#: ../plugins/system_upgrade.py:408
msgid "another upgrade tool is running. exiting quietly."
msgstr ""

#: ../plugins/system_upgrade.py:432
msgid "Rebooting to perform upgrade."
msgstr ""

#: ../plugins/system_upgrade.py:459 ../plugins/system_upgrade.py:465
msgid "Starting system upgrade. This will take a while."
msgstr ""

#: ../plugins/system_upgrade.py:496
msgid "Cleaning up downloaded data..."
msgstr ""

#: ../plugins/system_upgrade.py:527
msgid "Download finished."
msgstr ""

#: ../plugins/system_upgrade.py:531 ../plugins/system_upgrade.py:532
msgid "Upgrade complete! Cleaning up and rebooting..."
msgstr ""

#: ../plugins/debug.py:55
msgid "dump information about installed rpm packages to file"
msgstr ""

#: ../plugins/debug.py:69
msgid "do not attempt to dump the repository contents."
msgstr ""

#: ../plugins/debug.py:72
msgid "optional name of dump file"
msgstr ""

#: ../plugins/debug.py:97
#, python-format
msgid "Output written to: %s"
msgstr ""

#: ../plugins/debug.py:175
msgid "restore packages recorded in debug-dump file"
msgstr ""

#: ../plugins/debug.py:186
msgid "output commands that would be run to stdout."
msgstr ""

#: ../plugins/debug.py:189
msgid "Install the latest version of recorded packages."
msgstr ""

#: ../plugins/debug.py:192
msgid ""
"Ignore architecture and install missing packages matching the name, epoch, "
"version and release."
msgstr ""

#: ../plugins/debug.py:197
msgid "limit to specified type"
msgstr ""

#: ../plugins/debug.py:199
msgid "name of dump file"
msgstr ""

#: ../plugins/debug.py:268
#, python-format
msgid "Package %s is not available"
msgstr ""

#: ../plugins/debug.py:278
#, python-format
msgid "Bad dnf debug file: %s"
msgstr ""

#: ../plugins/local.py:115
msgid "'{}' is not a directory"
msgstr ""

#: ../plugins/local.py:124
msgid "Copying '{}' to local repo"
msgstr ""

#: ../plugins/local.py:130
msgid "Can't write file '{}'"
msgstr ""

#: ../plugins/local.py:145
msgid "Rebuilding local repo"
msgstr ""

#: ../plugins/tracer.py:75
msgid "For more information run:"
msgstr ""

#: ../plugins/tracer.py:102
msgid "Call to Tracer API ended unexpectedly:"
msgstr ""

#: ../plugins/tracer.py:104
msgid ""
"Please visit https://github.com/FrostyX/tracer/issues and submit the issue. "
"Thank you"
msgstr ""

#: ../plugins/tracer.py:106
msgid "We apologize for any inconvenience"
msgstr ""
